python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -399,7 +400,6 @@ def emit_with_mutating_variants(key, **kwargs):
     emit("aten::nll_loss_backward : (Tensor, Tensor, Tensor, Tensor?, int, int, Tensor) -> (Tensor)")
     emit("aten::bincount : (Tensor, Tensor?, int) -> (Tensor)")
     emit("aten::linalg_vector_norm : (Tensor, Scalar, int[]?, bool, int?) -> (Tensor)")
-    emit("aten::frobenius_norm.dim : (Tensor, int[], bool) -> (Tensor)")
```

Comment:
Did you remove this line before or after running `./build_tools/update_torch_ods.sh`? This is the line that generates the ODS for the op. Also, when I run thing locally on main, I don't see this op disappear.

Reply:
I hadn't removed anything; I didn't have any good reason to pick on frobenius :P

Perhaps it was something temporary with us running against head pytorch?  If it's working fine for you at head now I'll try clearing it all out and rerunning once I'm able to move over to the decomposition approach.

Reply:
Ah, maybe it was a temporary thing with PyTorch. There is a commit that makes changes to `frobenius_norm` that would appear when running `update_torch_ods.sh`: https://github.com/pytorch/pytorch/commit/1c0f0b33a0e013d6ec162cf488ff7643c4ffa33e

The commit has been merged into PyTorch and reverted twice, so this might come back in the future.